src/interfaces/package-export-transport.ts

```diff
@@ -60,8 +60,8 @@ export interface NodeExportTransport {
     serializedDocument?: Buffer;
 }
 
-export interface NodeSerializedContent {
-    variables: VariableDefinition[]
```

Comment:
Added it as an optional property [here](https://github.com/celonis/content-cli/pull/186/commits/14cb3ead3ee61a09ba67106fadaab277086660f5), because it's used when building nodes and packages in tests.